.htaccess

```diff
@@ -0,0 +1,31 @@
+# Save this file as UTF-8, or at least, plain ASCII text.
+# These are considered current best practice.
+# Don't change them unless you're having problems on your site.
+php_flag auto_detect_line_endings on
+php_value default_charset UTF-8
+
+# Change all below to reflect your needs
+
+# 'register_globals' was removed in PHP 5.4
+# Remove this line if you're using PHP >= 5.4.
+php_flag register_globals off
+
+# 'magic_quotes' were removed in PHP 7.
+# Remove these lines if you're using PHP >= 7.
+php_flag magic_quotes_gpc Off
+php_flag magic_quotes_runtime Off
+php_flag magic_quotes_sybase Off
+
+php_value date.default_latitude 43.133
+php_value date.default_longitude -115.6912
+php_value date.timezone America/Boise
```

Comment:
Yes, more comments. On all entries.
This one was supposed to be an example. Included with the release.
And on the admin page, (or maybe just the installer?), ask what the defaults should be and create the real file on the fly.
I intended for the system admin to set defaults for their site. Just once.
For instance, my ISP has timezone = UTC. I live in Idaho. Not very helpful.
If we could access GPS/location info, latitude/longitude, and track that in user_prefs, we could keep traveling boss/assistant timezones updated. Without trying to guess.